cms/plugin_pool.py

```diff
@@ -37,10 +37,67 @@ def clear(self):
         self.plugins = {}
         self.patched = False
 
+    def validate_templates(self, plugin=None):
+        """
+        Plugins templates are validated at this stage
+
```

Comment:
I think there's no reason for a blank line here. I'm not sure where that convention comes from.